src/plugins/LabelBot/strategies/integrationFromDocLink.ts

```diff
@@ -0,0 +1,14 @@
+import { IssueContext } from "../../../types";
```

Comment:
So should we introduce strategies or should it be a new "app" ?

Reply:
Because I think that both are really similar.

Reply:
It also means that we don't have to keep adding every event listener to labelbot as strategies require other events.

Reply:
Since LabelBot is so mangled in this PR with the split, it was easier to just start fresh <https://github.com/home-assistant/probot-home-assistant/pull/64>